Solutions/ESETPROTECT/Analytic Rules/ESETWebsiteBlocked.yaml

```diff
@@ -0,0 +1,38 @@
+id: 7b84fc5b-9ffb-4e9b-945b-5d480e330b3f
+name: Website blocked by ESET
+description: |
+  'Create alert on websites blocked by ESET.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: ESETPROTECT
+    dataTypes:
+      - Syslog
+queryFrequency: 5m
+queryPeriod: 5m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Exfiltration
+  - CommandAndControl
+  - InitialAccess
+query: |
+  ESETPROTECT
+  | where EventType == 'FilteredWebsites_Event'
+  | extend AccountCustomEntity = SrcUserName, URLCustomEntity = FilePath, HostCustomEntity = DvcHostname, IPCustomEntity = DvcIpAddr
+entityMappings:
+  - entityType: Account
+    fieldMappings:
+      - identifier: FullName
+        columnName: AccountCustomEntity
+  - entityType: Host
+    fieldMappings:
+      - identifier: FullName
+        columnName: HostCustomEntity
+  - entityType: IP
+    fieldMappings:
+      - identifier: Address
+        columnName: IPCustomEntity
+  - entityType: URL
+    fieldMappings:
+      - identifier: Url
+        columnName: URLCustomEntity
```

Comment:
@dmaasland, Please add field **Version** in Analytic rule file as mentioned below
"version: 1.0.0"
refer this link for reference. [Sample PR link](https://github.com/Azure/Azure-Sentinel/pull/2271)